RevenueCatUI/Data/Variables.swift

```diff
@@ -69,29 +70,33 @@ enum VariableHandler {
 @available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
 extension String {
 
-    func processed(with provider: VariableDataProvider) -> Self {
-        return VariableHandler.processVariables(in: self, with: provider)
+    func processed(with provider: VariableDataProvider, locale: Locale) -> Self {
+        return VariableHandler.processVariables(in: self, with: provider, locale: locale)
     }
 
 }
 
 @available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
 private extension VariableDataProvider {
 
-    func value(for variableName: String) -> String {
+    func value(for variableName: String, locale: Locale) -> String {
         switch variableName {
         case "price": return self.localizedPrice
         case "price_per_month": return self.localizedPricePerMonth
-        case "product_name": return self.productName
-        case "intro_duration":
-            guard let introDuration = self.introductoryOfferDuration else {
-                Logger.warning(
-                    "Unexpectedly tried to look for intro duration when there is none, this is a logic error."
-                )
-                return ""
+        case "total_price_and_per_month":
+            let price = self.localizedPrice
+            let perMonth = self.localizedPricePerMonth
+
+            if price == perMonth {
+                return price
+            } else {
+                let unit = Localization.abbreviatedUnitLocalizedString(for: .month, locale: locale)
+                return "\(price) (\(perMonth)/\(unit))"
```

Comment:
This will need better localization support (#2851 makes that easier, with the integrated `.xcstrings`) but this is a start.